src/extension.ts

```diff
@@ -74,10 +79,16 @@ function query(queryText: string): Promise<any> {
       query: queryText,
       location: config.get("location"),
       maximumBytesBilled: config.get("maximumBytesBilled"),
-      useLegacySql: config.get("useLegacySql")
+      useLegacySql: config.get("useLegacySql"),
+      dryRun: !!isDryRun
     })
     .then(data => {
       let job = data[0];
+      if (isDryRun) {
+        vscode.window.showInformationMessage(
+            `BigQuery dry run successful. Job ID: ${job.id}`);
```

Comment:
Might be worth re-using the template string below for consistency - and just appending "(dry run)" in this case.

Reply:
Done, thanks for the suggestion!